go.mod

```diff
@@ -3,10 +3,10 @@ module sigs.k8s.io/boskos
 go 1.18
 
 require (
-	github.com/IBM-Cloud/power-go-client v1.2.1
-	github.com/IBM/go-sdk-core/v5 v5.10.2
-	github.com/IBM/platform-services-go-sdk v0.28.6
-	github.com/IBM/vpc-go-sdk v0.27.0
+	github.com/IBM-Cloud/power-go-client v1.2.2
+	github.com/IBM/go-sdk-core/v5 v5.12.0
+	github.com/IBM/platform-services-go-sdk v0.31.3
+	github.com/IBM/vpc-go-sdk v0.30.0
```

Comment:
this update can go in a different PR to avoid dependency on the other maintainer for approving the code.